tests/ui/resolve/issue-105069.stderr

```diff
@@ -4,17 +4,19 @@ error[E0659]: `V` is ambiguous
 LL | use V;
    |     ^ ambiguous name
    |
-   = note: ambiguous because of multiple potential import sources
-note: `V` could refer to the variant imported here
+   = note: ambiguous because of multiple glob imports of a name in the same module
```

Comment:
Looks like you just fixed this FIXME - https://github.com/rust-lang/rust/pull/112086/files#diff-54ebaa0715c94a2acf974077e838aa584186e187f3d01edb1791fd71c78408caR937-R939 and we can now remove `AmbiguityKind::Import` in https://github.com/rust-lang/rust/pull/112086.